src/marqo/index.py

```diff
@@ -138,7 +138,7 @@ def search(self, q: str, searchable_attributes: Optional[List[str]] = None,
             "limit": limit,
             "searchMethod": search_method,
             "showHighlights": show_highlights,
-            "reranker": reranker,
+            "reRanker": reRanker,
```

Comment:
I believe this is all that needs to change
`"reRanker":reranker`

Reply:
For this, I will have to rename the parameter in the search function, and I think that will bring us back to square one.
what do you suggest @jn2clark

Reply:
Sorry, missed the next comment.
will change this.

Reply:
yeah, 
`"reRanker":reranker`
this is because this needs to match the api model in marqo (https://github.com/marqo-ai/marqo/blob/mainline/src/marqo/tensor_search/models/api_models.py#L20) which gets invoked here https://github.com/marqo-ai/marqo/blob/mainline/src/marqo/tensor_search/api.py#L111. hopefully that context helps a bit.

Reply:
Yes, makes sense.